reactive/easyrsa.py

```diff
@@ -414,8 +413,6 @@ def create_server_certificate(cn, san_list, name=None):
 
 def create_client_certificate(name='client'):
     '''Return a newly created client certificate and client key, by name.'''
-    client_cert = None
-    client_key = None
```

Comment:
`client_cert` and `client_key` variables do not need to be initialised here because they are later assigned values (non-conditionally)